# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1382925997 14400
#      Sun Oct 27 22:06:37 2013 -0400
# Node ID 9244d2a908fdd3e61e3f8fdba1d7fa8649caf5b9
# Parent  b5d58667d32dc61079dd98a2fe1d33bc659945ab
distribute options-usage.h

* libinterp/Makefile.am (octinclude_HEADERS):
List options-usage.h here.
(nodist_octinclude_HEADERS): Not here.

diff --git a/libinterp/Makefile.am b/libinterp/Makefile.am
--- a/libinterp/Makefile.am
+++ b/libinterp/Makefile.am
@@ -104,28 +104,28 @@ EXTRA_DIST = \
   $(BUILT_DISTFILES)
 
 octinclude_HEADERS = \
   corefcn/graphics-props.cc \
   parse-tree/oct-gperf.h \
   builtins.h \
   builtin-defun-decls.h \
   octave.h \
+  options-usage.h \
   $(OCTAVE_VALUE_INC) \
   $(PARSE_TREE_INC) \
   $(PARSER_INC) \
   $(OPERATORS_INC) \
   $(COREFCN_INC)
 
 nodist_octinclude_HEADERS = \
   corefcn/mxarray.h \
   corefcn/defaults.h \
   corefcn/graphics.h \
   oct-conf.h \
-  options-usage.h \
   version.h
 
 DIST_SRC = \
   octave.cc \
   version.cc \
   $(OCTAVE_VALUE_SRC) \
   $(PARSE_TREE_SRC) \
   $(PARSER_SRC) \
